src/cli.rs

```diff
@@ -829,7 +831,8 @@ impl clap::Args for Exec {
                        '{/}':  basename\n  \
                        '{//}': parent directory\n  \
                        '{.}':  path without file extension\n  \
-                       '{/.}': basename without file extension\n\n\
+                       '{/.}': basename without file extension\n  \
+                       '{{}':  literal '{' (for escaping)\n\n\
```

Comment:
is this still the old syntax? (comparing it with the above)

Reply:
Yeah, oops. Thanks for fixing that.